example_configs/flume_agent.yaml

```diff
@@ -0,0 +1,39 @@
+# Config to expose JMX-variables from Flume's Java proccess and map to Prometheus syntax.
+# The FlumeKafkaSink metrics are also exposed.
+#
+# When using Hortonworks and Ambari. You can use the Ambari > Flume > flume-env and attach the java-agent with the config to the flume process.
+# Proposed directory: /etc/jmx_exporter/
```

Comment:
He Brain, it's a comment which will be removed to be the same as other sample configs.